src/elements-experimental/journey-summary/journey-summary.visual.spec.ts

```diff
@@ -30,12 +31,14 @@ describe(`sbb-journey-summary`, () => {
     trip,
     tripBack,
     hasSlot,
+    a11yFootpath,
```

Comment:
If you like to test it, you need a case where a11yFootpath is true. Otherwise, you don't need the changes in this file

Reply:
attribute was removed by commit 7287e2fec65d62d72934ce26335ecc00b541829d